content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -0,0 +1,152 @@
+
+Use InfluxDB's Iceberg Exporter to convert and export your time-series data into the Iceberg table format.
+
+#### Example: Export data using the Iceberg Exporter
+
+```sh
+# Clone the Iceberg exporter repository
+git clone https://github.com/influxdata/influxdb_iox.git
+cd influxdb_iox/iceberg_exporter
+```
+
+- Configure the exporter with your InfluxDB data source and target Iceberg table location.
+- Run the exporter to generate Iceberg-compatible Parquet files.
+
+For more details, refer to the [Iceberg Exporter README](https://github.com/influxdata/influxdb_iox/tree/main/iceberg_exporter).
+
+### Step 4: Create an Iceberg table in Snowflake
+
+After exporting the data, create an Iceberg table in Snowflake.
+
+#### Example: Create an Iceberg table in Snowflake
+
+```sql
+CREATE ICEBERG TABLE my_iceberg_table
+  EXTERNAL_VOLUME = 'my_external_volume'
+  METADATA_FILE_PATH = 's3://my-bucket/path/to/metadata.json';
+```
+
+Ensure that `EXTERNAL_VOLUME` and `METADATA_FILE_PATH` point to your external storage and metadata file.
+
+### Step 5: Query the Iceberg table from Snowflake
+
+Once the Iceberg table is set up, you can query it using standard SQL in Snowflake.
+
+#### Example: Query the Iceberg table
+
+```sql
+SELECT * FROM my_iceberg_table
+WHERE timestamp > '2025-01-01';
+```
+
+## Interfaces for using Iceberg integration
+
+- **CLI**: `Influx CTL` enables users to trigger snapshot exports.
+- **API**: Provides REST endpoints to manage and configure snapshots.
+- **SQL (Snowflake)**: Query Iceberg tables using standard SQL.
+
+## CLI and API reference
+
+### CLI commands
+
+#### Example: Enable Iceberg feature and export a snapshot
+
+```sh
+# Enable Iceberg feature
+influxctl enable-iceberg
+
+# Export a snapshot
+influxctl export --namespace foo --table bar
+```
+
+### API endpoints
+
+#### Example: Export a snapshot
+
+- **Method**: `POST`
+- **Endpoint**: `/snapshots/export`
+- **Request body**:
+  
+```json
+{
+  "namespace": "foo",
+  "table": "bar"
+}
+```
+
+#### Example: Check snapshot status
+
+- **Method**: `GET`
+- **Endpoint**: `/snapshots/status`
+
+## Considerations and limitations
+
+- **Data consistency**: Ensure that the exported data in the Iceberg table is consistent with the source data in InfluxDB.
+- **Performance**: Query performance may vary based on data size and query complexity.
+- **Feature support**: Some advanced features of InfluxDB may not be fully supported in Snowflake through Iceberg integration.
+
+## Next steps
+
+- Enhance REST Catalog support.
+- Optimize AWS S3 access control and security.
+- Improve Grafana dashboards and alerting.
+- Expand compatibility testing with other Iceberg engines.
+
+## References
+
+- **InfluxDB Iceberg Exporter**: [GitHub Repository](https://github.com/influxdata/influxdb_iox/tree/main/iceberg_exporter)
+- **Snowflake Iceberg Tables**: [Snowflake Documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg)
```

Comment:
```suggestion
```

We don't want to forecast features or changes and you already have the links in context.